python_backend/api/v1/endpoints/documents.py

```
@@ -0,0 +1,111 @@
+        # 업로드된 디렉토리를 인덱싱
+        result = rag_service.ingest_documents(str(target_dir))
+        return {
+            "success": bool(result.get("success")),
+            "uploaded": saved,
+            "documents_processed": result.get("documents_processed", 0),
+            "index_path": str(target_dir),
+            "error": result.get("error") if not result.get("success") else None,
+        }
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)